src/Modules/Authentication/ResetPassword/ResetPassword.tsx

```diff
@@ -1,7 +1,292 @@
-import styles from './ResetPassword.module.css';
+import styles from "./ResetPassword.module.css";
+import { IconButton, InputAdornment, Stack } from "@mui/material";
+import image from "../../../assets/images/Reset-passwordIMG.jpg";
+
+import Typography from "@mui/material/Typography";
+import TextField from "@mui/material/TextField";
+import Button from "@mui/material/Button";
+import { Link, useNavigate } from "react-router-dom";
+import { useState } from "react";
+
+import { useForm, SubmitHandler } from "react-hook-form";
+import { toast } from "react-toastify";
+import {
+  AUTH_URL,
+  axiosInstanceAdmin,
+} from "../../../Services/END_POINTS/ADMIN/URLS";
+import {
+  EMAIL_VALIDATION,
+  PASWORD_VALIDATION,
+} from "../../../Services/Validation/VALIDATION";
+import { Visibility, VisibilityOff } from "@mui/icons-material";
+
+interface ResetPasswordFormData {
+  email: string;
+  seed: string;
+  password: string;
+  confirmPassword: string;
+}
 
 export default function ResetPassword() {
+  const [loading, setLoading] = useState<boolean>(false);
+  const navigate = useNavigate();
+  const [showPassword, setShowPassword] = useState<boolean>(false);
+  const [showConfirmPassword, setShowConfirmPassword] =
+    useState<boolean>(false);
+  // Hook Form setup with types
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+    getValues, // To validate confirm password
+  } = useForm<ResetPasswordFormData>();
+
+  // The function to handle form submission
+  const onSubmit: SubmitHandler<ResetPasswordFormData> = async (data) => {
+    setLoading(true);
+    try {
+      const response = await axiosInstanceAdmin.post(
+        AUTH_URL.RESET_PASSWORD,
+        data
+      );
+      toast.success(response.data.message);
```

Comment:
Since we are using the response, we need to provide the type to the post method `post<ResetResponse>`